lib/rules/function-url-quotes/index.js

```diff
@@ -86,16 +97,23 @@ function rule(expectation, options) {
 					return;
 				}
 
+				// @ts-expect-error -- Should the message include the function name?
 				complain(messages.expected(functionName), node, complaintIndex);
```

Comment:
[question] The passed `functionName` is ignored now:

https://github.com/stylelint/stylelint/blob/544d85763f2a36ac83aa5d924e41713b1177d3bf/lib/rules/function-url-quotes/index.js#L15-L18

If we don't need to change the messages, I'll remove the `functionName` variable.

On the other hand, if we would improve the message like `Expected quotes for "url" function`, I'll change the message and test with another PR.

What do you think?

Reply:
Let's improve the message to `Expected quotes around "${functionName}" function argument` to be consistent with the messages in the other `*-quote` rules ([spec ref for "argument"](https://drafts.csswg.org/css-values/#functional-notation)).

---

Incidentally, it was interesting to [see that a `src()` function may be coming to the language](https://drafts.csswg.org/css-values/#funcdef-src) so that `var()` can be used.

Reply:
Thanks for the advice. I'll create a PR to improve the message! 👍🏼 

> Incidentally, it was interesting to [see that a `src()` function may be coming to the language](https://drafts.csswg.org/css-values/#funcdef-src) so that `var()` can be used.

Oh, I also didn't know the feature. It would be more useful! 😄

Reply:
I've opened PR #5488.

Reply:
Thanks. I've merged it. Feel free to rebase this pull request, remove the `@ts-expect-error` and merge.

Reply:
Thanks! I've removed `@ts-expect-error` comments via 5bbae44b. When the CI passes, I'll merge it.